tests/desktop/test_paypal.py

```diff
@@ -13,6 +13,7 @@ class TestPaypal:
 
     addon_name = 'Firebug'
 
+    @pytest.mark.flaky(reruns=1)
```

Comment:
1 rerun is plenty i guess? and what's the expectation here? if this fails after the rerun we'll eventually remove it, as in no plans to fix the flakiness?